src/components/pages/Theme/Theme.jsx

```diff
@@ -159,6 +160,16 @@ export const Theme = () => {
         </Segment>
       </div>
 
+      <div className='section'>
+        <Segment>
+          <Header as='h2' dividing>
+            Divider
+          </Header>
+
+          <Divider />
```

Comment:
Let's add some text above and below the divider to demonstrate it in context.

![Screenshot 2023-06-07 at 4 08 16 PM](https://github.com/abby-moffitt/GA-Semantic/assets/1568524/97c6fd62-cda8-4d34-baec-b4cc091727a3)